### PR TITLE
Added TrustedWire to http request

### PR DESCRIPTION
**Pull request by SherifWaly**
PR for #29

### CODE REVIEW

**Comment by SherifWaly** [comment]
@amihaiemil Please check this commit :) 
When I called ``trusted()`` at ``RtServicesITCase`` the test passes now but I'm now sure whether that was the required or not.

**Comment by amihaiemil** [comment]
> When I called trusted() at RtServicesITCase the test passes now but I'm now sure whether that was the required or not.

@SherifWaly But have you tried to run the test without calling the method? Then it should fail :D Nevertheless, leave a puzzle in the code, I'll write some tests (somehow) for it -- maybe with MkGrizzly server. We start an in-memory server, make a request and see if it reached the server with certain HTTP headers :D

File: src/main/java/com/amihaiemil/versioneye/RtVersionEye.java [hunk 1]
```diff
@@ -82,5 +85,12 @@ public Services services() {
     public Users users() {
         return new RtUsers(this.entry);
     }
-
+    
+    @Override
+    public VersionEye trusted() throws IOException {
+        return new RtVersionEye(
+            this.entry.method(Request.GET).through(TrustedWire.class)
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/versioneye/RtVersionEye.java]: @SherifWaly Is it mandatory to set the ``GET`` method before calling ``through(...)``? It is by default ``GET`` so it makes no sense to set it.

File: src/main/java/com/amihaiemil/versioneye/VersionEye.java [hunk 2]
```diff
@@ -47,4 +49,11 @@
      * @return Users.
      */
     Users users();
+    
+    /**
+     * VersionEye with trusted wire.
+     * @return VersionEye.
+     * @throws IOException
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/versioneye/VersionEye.java]: @SherifWaly ``@throws IOException If something goes wrong with the HTTP call`` and can you also add ``@see http://http.jcabi.com/pkix-validator.html``, so the user can go and read why he might need this method :D

**Review by amihaiemil — COMMENTED** [review]
@SherifWaly Looks good - just a few comments :)

**Comment by SherifWaly** [comment]
@amihaiemil I've tried running the test without calling ``trusted()`` and the test was failing but it passed after adding TrustedWire :D

**Comment by amihaiemil** [comment]
@SherifWaly Looks nice! :)

**Pull request merged**